examples/coappearances/coappearances.flatdata

```diff
@@ -129,5 +129,26 @@ archive Graph {
 
     // All strings contained in the data separated by '\0'.
     strings: raw_data;
+
+    // Optional archive containing calculated statistics.
+    @optional
+    calculated_data : archive CalculatedData;
+}
+
+struct Invariants {
+    max_degree : u32 : 16;
+    max_degree_ref : u32 : 16;
+    min_degree : u32 : 16;
+    min_degree_ref : u32 : 16;
+    num_connected_components : u32 : 16;
+}
+
+struct Degree {
+    value : u32 : 16;
+}
+
+archive CalculatedData {
+    invariants : Invariants;
+    vertex_degrees : vector< Degree >;
```

Comment:
How is this indexed? -> document

Reply:
Added docs and implicit bound into the parent archive.